[2024-04-03T16:20:34.579+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Chuksfx_ETL.transform_response scheduled__2024-04-02T00:00:00+00:00 [queued]>
[2024-04-03T16:20:34.598+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Chuksfx_ETL.transform_response scheduled__2024-04-02T00:00:00+00:00 [queued]>
[2024-04-03T16:20:34.602+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 0
[2024-04-03T16:20:34.633+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): transform_response> on 2024-04-02 00:00:00+00:00
[2024-04-03T16:20:34.644+0000] {standard_task_runner.py:60} INFO - Started process 2999 to run task
[2024-04-03T16:20:34.649+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'Chuksfx_ETL', 'transform_response', 'scheduled__2024-04-02T00:00:00+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/include/fx_dag.py', '--cfg-path', '/tmp/tmp5svy55zp']
[2024-04-03T16:20:34.652+0000] {standard_task_runner.py:88} INFO - Job 14: Subtask transform_response
[2024-04-03T16:20:34.745+0000] {task_command.py:423} INFO - Running <TaskInstance: Chuksfx_ETL.transform_response scheduled__2024-04-02T00:00:00+00:00 [running]> on host 506fa6827c3e
[2024-04-03T16:20:34.892+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Chuks' AIRFLOW_CTX_DAG_ID='Chuksfx_ETL' AIRFLOW_CTX_TASK_ID='transform_response' AIRFLOW_CTX_EXECUTION_DATE='2024-04-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-02T00:00:00+00:00'
[2024-04-03T16:20:34.896+0000] {taskinstance.py:2731} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 444, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 414, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 200, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 217, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/include/transform.py", line 15, in transform_data
    timestamp = response['timestamp']
KeyError: 'timestamp'
[2024-04-03T16:20:34.920+0000] {taskinstance.py:1149} INFO - Marking task as FAILED. dag_id=Chuksfx_ETL, task_id=transform_response, execution_date=20240402T000000, start_date=20240403T162034, end_date=20240403T162034
[2024-04-03T16:20:34.949+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 14 for task transform_response ('timestamp'; 2999)
[2024-04-03T16:20:34.986+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-04-03T16:29:46.653+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Chuksfx_ETL.transform_response scheduled__2024-04-02T00:00:00+00:00 [queued]>
[2024-04-03T16:29:46.671+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Chuksfx_ETL.transform_response scheduled__2024-04-02T00:00:00+00:00 [queued]>
[2024-04-03T16:29:46.672+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 0
[2024-04-03T16:29:46.698+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): transform_response> on 2024-04-02 00:00:00+00:00
[2024-04-03T16:29:46.708+0000] {standard_task_runner.py:60} INFO - Started process 3141 to run task
[2024-04-03T16:29:46.714+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'Chuksfx_ETL', 'transform_response', 'scheduled__2024-04-02T00:00:00+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/include/fx_dag.py', '--cfg-path', '/tmp/tmpnlhiut0m']
[2024-04-03T16:29:46.716+0000] {standard_task_runner.py:88} INFO - Job 16: Subtask transform_response
[2024-04-03T16:29:46.803+0000] {task_command.py:423} INFO - Running <TaskInstance: Chuksfx_ETL.transform_response scheduled__2024-04-02T00:00:00+00:00 [running]> on host 506fa6827c3e
[2024-04-03T16:29:46.929+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Chuks' AIRFLOW_CTX_DAG_ID='Chuksfx_ETL' AIRFLOW_CTX_TASK_ID='transform_response' AIRFLOW_CTX_EXECUTION_DATE='2024-04-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-02T00:00:00+00:00'
[2024-04-03T16:29:46.937+0000] {taskinstance.py:2731} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 444, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 414, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 200, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 217, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/include/transform.py", line 15, in transform_data
    timestamp = response['timestamp']
KeyError: 'timestamp'
[2024-04-03T16:29:46.960+0000] {taskinstance.py:1149} INFO - Marking task as FAILED. dag_id=Chuksfx_ETL, task_id=transform_response, execution_date=20240402T000000, start_date=20240403T162946, end_date=20240403T162946
[2024-04-03T16:29:46.979+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 16 for task transform_response ('timestamp'; 3141)
[2024-04-03T16:29:47.008+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-04-03T16:52:49.661+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Chuksfx_ETL.transform_response scheduled__2024-04-02T00:00:00+00:00 [queued]>
[2024-04-03T16:52:49.672+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Chuksfx_ETL.transform_response scheduled__2024-04-02T00:00:00+00:00 [queued]>
[2024-04-03T16:52:49.674+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 0
[2024-04-03T16:52:49.692+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): transform_response> on 2024-04-02 00:00:00+00:00
[2024-04-03T16:52:49.700+0000] {standard_task_runner.py:60} INFO - Started process 3491 to run task
[2024-04-03T16:52:49.704+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'Chuksfx_ETL', 'transform_response', 'scheduled__2024-04-02T00:00:00+00:00', '--job-id', '18', '--raw', '--subdir', 'DAGS_FOLDER/include/fx_dag.py', '--cfg-path', '/tmp/tmpd0x_78nw']
[2024-04-03T16:52:49.705+0000] {standard_task_runner.py:88} INFO - Job 18: Subtask transform_response
[2024-04-03T16:52:49.767+0000] {task_command.py:423} INFO - Running <TaskInstance: Chuksfx_ETL.transform_response scheduled__2024-04-02T00:00:00+00:00 [running]> on host 506fa6827c3e
[2024-04-03T16:52:49.846+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Chuks' AIRFLOW_CTX_DAG_ID='Chuksfx_ETL' AIRFLOW_CTX_TASK_ID='transform_response' AIRFLOW_CTX_EXECUTION_DATE='2024-04-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-02T00:00:00+00:00'
[2024-04-03T16:52:49.849+0000] {taskinstance.py:2731} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 444, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 414, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 200, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 217, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/include/transform.py", line 15, in transform_data
    timestamp = response['timestamp']
KeyError: 'timestamp'
[2024-04-03T16:52:49.863+0000] {taskinstance.py:1149} INFO - Marking task as FAILED. dag_id=Chuksfx_ETL, task_id=transform_response, execution_date=20240402T000000, start_date=20240403T165249, end_date=20240403T165249
[2024-04-03T16:52:49.877+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 18 for task transform_response ('timestamp'; 3491)
[2024-04-03T16:52:49.917+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-04-03T16:55:12.529+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Chuksfx_ETL.transform_response scheduled__2024-04-02T00:00:00+00:00 [queued]>
[2024-04-03T16:55:12.540+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Chuksfx_ETL.transform_response scheduled__2024-04-02T00:00:00+00:00 [queued]>
[2024-04-03T16:55:12.541+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 0
[2024-04-03T16:55:12.556+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): transform_response> on 2024-04-02 00:00:00+00:00
[2024-04-03T16:55:12.563+0000] {standard_task_runner.py:60} INFO - Started process 3530 to run task
[2024-04-03T16:55:12.566+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'Chuksfx_ETL', 'transform_response', 'scheduled__2024-04-02T00:00:00+00:00', '--job-id', '20', '--raw', '--subdir', 'DAGS_FOLDER/include/fx_dag.py', '--cfg-path', '/tmp/tmpyj6ny3jb']
[2024-04-03T16:55:12.567+0000] {standard_task_runner.py:88} INFO - Job 20: Subtask transform_response
[2024-04-03T16:55:12.630+0000] {task_command.py:423} INFO - Running <TaskInstance: Chuksfx_ETL.transform_response scheduled__2024-04-02T00:00:00+00:00 [running]> on host 506fa6827c3e
[2024-04-03T16:55:12.713+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Chuks' AIRFLOW_CTX_DAG_ID='Chuksfx_ETL' AIRFLOW_CTX_TASK_ID='transform_response' AIRFLOW_CTX_EXECUTION_DATE='2024-04-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-02T00:00:00+00:00'
[2024-04-03T16:55:12.715+0000] {taskinstance.py:2731} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 444, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 414, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 200, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 217, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/include/transform.py", line 15, in transform_data
    timestamp = response['timestamp']
KeyError: 'timestamp'
[2024-04-03T16:55:12.727+0000] {taskinstance.py:1149} INFO - Marking task as FAILED. dag_id=Chuksfx_ETL, task_id=transform_response, execution_date=20240402T000000, start_date=20240403T165512, end_date=20240403T165512
[2024-04-03T16:55:12.741+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 20 for task transform_response ('timestamp'; 3530)
[2024-04-03T16:55:12.779+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-04-03T17:20:33.414+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Chuksfx_ETL.transform_response scheduled__2024-04-02T00:00:00+00:00 [queued]>
[2024-04-03T17:20:33.424+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Chuksfx_ETL.transform_response scheduled__2024-04-02T00:00:00+00:00 [queued]>
[2024-04-03T17:20:33.425+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 0
[2024-04-03T17:20:33.437+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): transform_response> on 2024-04-02 00:00:00+00:00
[2024-04-03T17:20:33.444+0000] {standard_task_runner.py:60} INFO - Started process 3917 to run task
[2024-04-03T17:20:33.448+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'Chuksfx_ETL', 'transform_response', 'scheduled__2024-04-02T00:00:00+00:00', '--job-id', '24', '--raw', '--subdir', 'DAGS_FOLDER/include/fx_dag.py', '--cfg-path', '/tmp/tmp72g7im8_']
[2024-04-03T17:20:33.450+0000] {standard_task_runner.py:88} INFO - Job 24: Subtask transform_response
[2024-04-03T17:20:33.504+0000] {task_command.py:423} INFO - Running <TaskInstance: Chuksfx_ETL.transform_response scheduled__2024-04-02T00:00:00+00:00 [running]> on host 506fa6827c3e
[2024-04-03T17:20:33.587+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Chuks' AIRFLOW_CTX_DAG_ID='Chuksfx_ETL' AIRFLOW_CTX_TASK_ID='transform_response' AIRFLOW_CTX_EXECUTION_DATE='2024-04-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-02T00:00:00+00:00'
[2024-04-03T17:20:33.589+0000] {taskinstance.py:2731} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 444, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 414, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 200, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 217, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/include/transform.py", line 15, in transform_data
    timestamp = response['timestamp']
KeyError: 'timestamp'
[2024-04-03T17:20:33.602+0000] {taskinstance.py:1149} INFO - Marking task as FAILED. dag_id=Chuksfx_ETL, task_id=transform_response, execution_date=20240402T000000, start_date=20240403T172033, end_date=20240403T172033
[2024-04-03T17:20:33.617+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 24 for task transform_response ('timestamp'; 3917)
[2024-04-03T17:20:33.661+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-04-03T17:35:05.277+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Chuksfx_ETL.transform_response scheduled__2024-04-02T00:00:00+00:00 [queued]>
[2024-04-03T17:35:05.287+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Chuksfx_ETL.transform_response scheduled__2024-04-02T00:00:00+00:00 [queued]>
[2024-04-03T17:35:05.288+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 0
[2024-04-03T17:35:05.301+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): transform_response> on 2024-04-02 00:00:00+00:00
[2024-04-03T17:35:05.307+0000] {standard_task_runner.py:60} INFO - Started process 4131 to run task
[2024-04-03T17:35:05.309+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'Chuksfx_ETL', 'transform_response', 'scheduled__2024-04-02T00:00:00+00:00', '--job-id', '26', '--raw', '--subdir', 'DAGS_FOLDER/include/fx_dag.py', '--cfg-path', '/tmp/tmpjhgd0e5p']
[2024-04-03T17:35:05.311+0000] {standard_task_runner.py:88} INFO - Job 26: Subtask transform_response
[2024-04-03T17:35:05.352+0000] {task_command.py:423} INFO - Running <TaskInstance: Chuksfx_ETL.transform_response scheduled__2024-04-02T00:00:00+00:00 [running]> on host 506fa6827c3e
[2024-04-03T17:35:05.431+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Chuks' AIRFLOW_CTX_DAG_ID='Chuksfx_ETL' AIRFLOW_CTX_TASK_ID='transform_response' AIRFLOW_CTX_EXECUTION_DATE='2024-04-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-02T00:00:00+00:00'
[2024-04-03T17:35:05.433+0000] {taskinstance.py:2731} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 444, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 414, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 200, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 217, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/include/transform.py", line 15, in transform_data
    timestamp = response['timestamp']
KeyError: 'timestamp'
[2024-04-03T17:35:05.445+0000] {taskinstance.py:1149} INFO - Marking task as FAILED. dag_id=Chuksfx_ETL, task_id=transform_response, execution_date=20240402T000000, start_date=20240403T173505, end_date=20240403T173505
[2024-04-03T17:35:05.457+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 26 for task transform_response ('timestamp'; 4131)
[2024-04-03T17:35:05.483+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-04-03T17:39:23.424+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Chuksfx_ETL.transform_response scheduled__2024-04-02T00:00:00+00:00 [queued]>
[2024-04-03T17:39:23.435+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Chuksfx_ETL.transform_response scheduled__2024-04-02T00:00:00+00:00 [queued]>
[2024-04-03T17:39:23.436+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 0
[2024-04-03T17:39:23.457+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): transform_response> on 2024-04-02 00:00:00+00:00
[2024-04-03T17:39:23.466+0000] {standard_task_runner.py:60} INFO - Started process 4201 to run task
[2024-04-03T17:39:23.473+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'Chuksfx_ETL', 'transform_response', 'scheduled__2024-04-02T00:00:00+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/include/fx_dag.py', '--cfg-path', '/tmp/tmp6kjpnnzw']
[2024-04-03T17:39:23.474+0000] {standard_task_runner.py:88} INFO - Job 28: Subtask transform_response
[2024-04-03T17:39:23.524+0000] {task_command.py:423} INFO - Running <TaskInstance: Chuksfx_ETL.transform_response scheduled__2024-04-02T00:00:00+00:00 [running]> on host 506fa6827c3e
[2024-04-03T17:39:23.614+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Chuks' AIRFLOW_CTX_DAG_ID='Chuksfx_ETL' AIRFLOW_CTX_TASK_ID='transform_response' AIRFLOW_CTX_EXECUTION_DATE='2024-04-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-02T00:00:00+00:00'
[2024-04-03T17:39:23.618+0000] {taskinstance.py:2731} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 444, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 414, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 200, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 217, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/include/transform.py", line 15, in transform_data
    timestamp = response['timestamp']
KeyError: 'timestamp'
[2024-04-03T17:39:23.634+0000] {taskinstance.py:1149} INFO - Marking task as FAILED. dag_id=Chuksfx_ETL, task_id=transform_response, execution_date=20240402T000000, start_date=20240403T173923, end_date=20240403T173923
[2024-04-03T17:39:23.655+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 28 for task transform_response ('timestamp'; 4201)
[2024-04-03T17:39:23.686+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-04-03T17:55:32.231+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Chuksfx_ETL.transform_response scheduled__2024-04-02T00:00:00+00:00 [queued]>
[2024-04-03T17:55:32.242+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Chuksfx_ETL.transform_response scheduled__2024-04-02T00:00:00+00:00 [queued]>
[2024-04-03T17:55:32.243+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 0
[2024-04-03T17:55:32.262+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): transform_response> on 2024-04-02 00:00:00+00:00
[2024-04-03T17:55:32.274+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'Chuksfx_ETL', 'transform_response', 'scheduled__2024-04-02T00:00:00+00:00', '--job-id', '32', '--raw', '--subdir', 'DAGS_FOLDER/include/fx_dag.py', '--cfg-path', '/tmp/tmpd8ruqnmt']
[2024-04-03T17:55:32.276+0000] {standard_task_runner.py:88} INFO - Job 32: Subtask transform_response
[2024-04-03T17:55:32.270+0000] {standard_task_runner.py:60} INFO - Started process 4454 to run task
[2024-04-03T17:55:32.333+0000] {task_command.py:423} INFO - Running <TaskInstance: Chuksfx_ETL.transform_response scheduled__2024-04-02T00:00:00+00:00 [running]> on host 506fa6827c3e
[2024-04-03T17:55:32.411+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Chuks' AIRFLOW_CTX_DAG_ID='Chuksfx_ETL' AIRFLOW_CTX_TASK_ID='transform_response' AIRFLOW_CTX_EXECUTION_DATE='2024-04-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-02T00:00:00+00:00'
[2024-04-03T17:55:32.446+0000] {python.py:202} INFO - Done. Returned value was: None
[2024-04-03T17:55:32.458+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=Chuksfx_ETL, task_id=transform_response, execution_date=20240402T000000, start_date=20240403T175532, end_date=20240403T175532
[2024-04-03T17:55:32.493+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-03T18:00:38.806+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Chuksfx_ETL.transform_response scheduled__2024-04-02T00:00:00+00:00 [queued]>
[2024-04-03T18:00:38.816+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Chuksfx_ETL.transform_response scheduled__2024-04-02T00:00:00+00:00 [queued]>
[2024-04-03T18:00:38.817+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 0
[2024-04-03T18:00:38.831+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): transform_response> on 2024-04-02 00:00:00+00:00
[2024-04-03T18:00:38.839+0000] {standard_task_runner.py:60} INFO - Started process 4543 to run task
[2024-04-03T18:00:38.843+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'Chuksfx_ETL', 'transform_response', 'scheduled__2024-04-02T00:00:00+00:00', '--job-id', '35', '--raw', '--subdir', 'DAGS_FOLDER/include/fx_dag.py', '--cfg-path', '/tmp/tmp47p0mmif']
[2024-04-03T18:00:38.845+0000] {standard_task_runner.py:88} INFO - Job 35: Subtask transform_response
[2024-04-03T18:00:38.894+0000] {task_command.py:423} INFO - Running <TaskInstance: Chuksfx_ETL.transform_response scheduled__2024-04-02T00:00:00+00:00 [running]> on host 506fa6827c3e
[2024-04-03T18:00:38.966+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Chuks' AIRFLOW_CTX_DAG_ID='Chuksfx_ETL' AIRFLOW_CTX_TASK_ID='transform_response' AIRFLOW_CTX_EXECUTION_DATE='2024-04-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-02T00:00:00+00:00'
[2024-04-03T18:00:38.986+0000] {python.py:202} INFO - Done. Returned value was: None
[2024-04-03T18:00:38.996+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=Chuksfx_ETL, task_id=transform_response, execution_date=20240402T000000, start_date=20240403T180038, end_date=20240403T180038
[2024-04-03T18:00:39.056+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-03T19:44:03.056+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Chuksfx_ETL.transform_response scheduled__2024-04-02T00:00:00+00:00 [queued]>
[2024-04-03T19:44:03.063+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Chuksfx_ETL.transform_response scheduled__2024-04-02T00:00:00+00:00 [queued]>
[2024-04-03T19:44:03.064+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 0
[2024-04-03T19:44:03.075+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): transform_response> on 2024-04-02 00:00:00+00:00
[2024-04-03T19:44:03.080+0000] {standard_task_runner.py:60} INFO - Started process 6079 to run task
[2024-04-03T19:44:03.083+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'Chuksfx_ETL', 'transform_response', 'scheduled__2024-04-02T00:00:00+00:00', '--job-id', '38', '--raw', '--subdir', 'DAGS_FOLDER/include/fx_dag.py', '--cfg-path', '/tmp/tmpbklo0hlg']
[2024-04-03T19:44:03.084+0000] {standard_task_runner.py:88} INFO - Job 38: Subtask transform_response
[2024-04-03T19:44:03.122+0000] {task_command.py:423} INFO - Running <TaskInstance: Chuksfx_ETL.transform_response scheduled__2024-04-02T00:00:00+00:00 [running]> on host 506fa6827c3e
[2024-04-03T19:44:03.174+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Chuks' AIRFLOW_CTX_DAG_ID='Chuksfx_ETL' AIRFLOW_CTX_TASK_ID='transform_response' AIRFLOW_CTX_EXECUTION_DATE='2024-04-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-02T00:00:00+00:00'
[2024-04-03T19:44:03.185+0000] {python.py:202} INFO - Done. Returned value was: None
[2024-04-03T19:44:03.193+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=Chuksfx_ETL, task_id=transform_response, execution_date=20240402T000000, start_date=20240403T194403, end_date=20240403T194403
[2024-04-03T19:44:03.216+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-03T19:44:03.226+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/models/baseoperator.py:1201: AirflowProviderDeprecationWarning: Call to deprecated class SnowflakeOperator. (This class is deprecated. Please use `***.providers.common.sql.operators.sql.SQLExecuteQueryOperator`. Also, you can provide `hook_params={'warehouse': <warehouse>, 'database': <database>, 'role': <role>, 'schema': <schema>, 'authenticator': <authenticator>,'session_parameters': <session_parameters>}`.)
  result = cls.__new__(cls)

[2024-04-03T19:44:03.237+0000] {taskinstance.py:3312} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-04-03T19:46:40.103+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Chuksfx_ETL.transform_response scheduled__2024-04-02T00:00:00+00:00 [queued]>
[2024-04-03T19:46:40.111+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Chuksfx_ETL.transform_response scheduled__2024-04-02T00:00:00+00:00 [queued]>
[2024-04-03T19:46:40.111+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 0
[2024-04-03T19:46:40.121+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): transform_response> on 2024-04-02 00:00:00+00:00
[2024-04-03T19:46:40.127+0000] {standard_task_runner.py:60} INFO - Started process 6128 to run task
[2024-04-03T19:46:40.129+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'Chuksfx_ETL', 'transform_response', 'scheduled__2024-04-02T00:00:00+00:00', '--job-id', '41', '--raw', '--subdir', 'DAGS_FOLDER/include/fx_dag.py', '--cfg-path', '/tmp/tmp2k3p27fp']
[2024-04-03T19:46:40.130+0000] {standard_task_runner.py:88} INFO - Job 41: Subtask transform_response
[2024-04-03T19:46:40.167+0000] {task_command.py:423} INFO - Running <TaskInstance: Chuksfx_ETL.transform_response scheduled__2024-04-02T00:00:00+00:00 [running]> on host 506fa6827c3e
[2024-04-03T19:46:40.226+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Chuks' AIRFLOW_CTX_DAG_ID='Chuksfx_ETL' AIRFLOW_CTX_TASK_ID='transform_response' AIRFLOW_CTX_EXECUTION_DATE='2024-04-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-02T00:00:00+00:00'
[2024-04-03T19:46:40.234+0000] {python.py:202} INFO - Done. Returned value was: None
[2024-04-03T19:46:40.242+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=Chuksfx_ETL, task_id=transform_response, execution_date=20240402T000000, start_date=20240403T194640, end_date=20240403T194640
[2024-04-03T19:46:40.261+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-03T19:46:40.270+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/models/baseoperator.py:1201: AirflowProviderDeprecationWarning: Call to deprecated class SnowflakeOperator. (This class is deprecated. Please use `***.providers.common.sql.operators.sql.SQLExecuteQueryOperator`. Also, you can provide `hook_params={'warehouse': <warehouse>, 'database': <database>, 'role': <role>, 'schema': <schema>, 'authenticator': <authenticator>,'session_parameters': <session_parameters>}`.)
  result = cls.__new__(cls)

[2024-04-03T19:46:40.282+0000] {taskinstance.py:3312} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-04-03T20:03:27.672+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Chuksfx_ETL.transform_response scheduled__2024-04-02T00:00:00+00:00 [queued]>
[2024-04-03T20:03:27.680+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Chuksfx_ETL.transform_response scheduled__2024-04-02T00:00:00+00:00 [queued]>
[2024-04-03T20:03:27.680+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 0
[2024-04-03T20:03:27.690+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): transform_response> on 2024-04-02 00:00:00+00:00
[2024-04-03T20:03:27.695+0000] {standard_task_runner.py:60} INFO - Started process 6399 to run task
[2024-04-03T20:03:27.698+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'Chuksfx_ETL', 'transform_response', 'scheduled__2024-04-02T00:00:00+00:00', '--job-id', '45', '--raw', '--subdir', 'DAGS_FOLDER/include/fx_dag.py', '--cfg-path', '/tmp/tmpo0qe3x18']
[2024-04-03T20:03:27.699+0000] {standard_task_runner.py:88} INFO - Job 45: Subtask transform_response
[2024-04-03T20:03:27.739+0000] {task_command.py:423} INFO - Running <TaskInstance: Chuksfx_ETL.transform_response scheduled__2024-04-02T00:00:00+00:00 [running]> on host 506fa6827c3e
[2024-04-03T20:03:27.791+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Chuks' AIRFLOW_CTX_DAG_ID='Chuksfx_ETL' AIRFLOW_CTX_TASK_ID='transform_response' AIRFLOW_CTX_EXECUTION_DATE='2024-04-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-02T00:00:00+00:00'
[2024-04-03T20:03:27.800+0000] {python.py:202} INFO - Done. Returned value was: None
[2024-04-03T20:03:27.807+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=Chuksfx_ETL, task_id=transform_response, execution_date=20240402T000000, start_date=20240403T200327, end_date=20240403T200327
[2024-04-03T20:03:27.832+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-03T20:03:27.839+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/models/baseoperator.py:1201: AirflowProviderDeprecationWarning: Call to deprecated class SnowflakeOperator. (This class is deprecated. Please use `***.providers.common.sql.operators.sql.SQLExecuteQueryOperator`. Also, you can provide `hook_params={'warehouse': <warehouse>, 'database': <database>, 'role': <role>, 'schema': <schema>, 'authenticator': <authenticator>,'session_parameters': <session_parameters>}`.)
  result = cls.__new__(cls)

[2024-04-03T20:03:27.852+0000] {taskinstance.py:3312} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-04-03T20:37:20.757+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Chuksfx_ETL.transform_response scheduled__2024-04-02T00:00:00+00:00 [queued]>
[2024-04-03T20:37:20.764+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Chuksfx_ETL.transform_response scheduled__2024-04-02T00:00:00+00:00 [queued]>
[2024-04-03T20:37:20.765+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 0
[2024-04-03T20:37:20.774+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): transform_response> on 2024-04-02 00:00:00+00:00
[2024-04-03T20:37:20.778+0000] {standard_task_runner.py:60} INFO - Started process 6931 to run task
[2024-04-03T20:37:20.780+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'Chuksfx_ETL', 'transform_response', 'scheduled__2024-04-02T00:00:00+00:00', '--job-id', '50', '--raw', '--subdir', 'DAGS_FOLDER/include/fx_dag.py', '--cfg-path', '/tmp/tmprxiooz_f']
[2024-04-03T20:37:20.781+0000] {standard_task_runner.py:88} INFO - Job 50: Subtask transform_response
[2024-04-03T20:37:20.819+0000] {task_command.py:423} INFO - Running <TaskInstance: Chuksfx_ETL.transform_response scheduled__2024-04-02T00:00:00+00:00 [running]> on host 506fa6827c3e
[2024-04-03T20:37:20.869+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Chuks' AIRFLOW_CTX_DAG_ID='Chuksfx_ETL' AIRFLOW_CTX_TASK_ID='transform_response' AIRFLOW_CTX_EXECUTION_DATE='2024-04-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-02T00:00:00+00:00'
[2024-04-03T20:37:20.877+0000] {python.py:202} INFO - Done. Returned value was: None
[2024-04-03T20:37:20.884+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=Chuksfx_ETL, task_id=transform_response, execution_date=20240402T000000, start_date=20240403T203720, end_date=20240403T203720
[2024-04-03T20:37:20.914+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-03T20:37:20.926+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/models/baseoperator.py:1201: AirflowProviderDeprecationWarning: Call to deprecated class SnowflakeOperator. (This class is deprecated. Please use `***.providers.common.sql.operators.sql.SQLExecuteQueryOperator`. Also, you can provide `hook_params={'warehouse': <warehouse>, 'database': <database>, 'role': <role>, 'schema': <schema>, 'authenticator': <authenticator>,'session_parameters': <session_parameters>}`.)
  result = cls.__new__(cls)

[2024-04-03T20:37:20.938+0000] {taskinstance.py:3312} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-04-03T21:22:48.616+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Chuksfx_ETL.transform_response scheduled__2024-04-02T00:00:00+00:00 [queued]>
[2024-04-03T21:22:48.625+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Chuksfx_ETL.transform_response scheduled__2024-04-02T00:00:00+00:00 [queued]>
[2024-04-03T21:22:48.625+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 0
[2024-04-03T21:22:48.638+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): transform_response> on 2024-04-02 00:00:00+00:00
[2024-04-03T21:22:48.643+0000] {standard_task_runner.py:60} INFO - Started process 7631 to run task
[2024-04-03T21:22:48.646+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'Chuksfx_ETL', 'transform_response', 'scheduled__2024-04-02T00:00:00+00:00', '--job-id', '53', '--raw', '--subdir', 'DAGS_FOLDER/include/fx_dag.py', '--cfg-path', '/tmp/tmpntayk2vw']
[2024-04-03T21:22:48.647+0000] {standard_task_runner.py:88} INFO - Job 53: Subtask transform_response
[2024-04-03T21:22:48.687+0000] {task_command.py:423} INFO - Running <TaskInstance: Chuksfx_ETL.transform_response scheduled__2024-04-02T00:00:00+00:00 [running]> on host 506fa6827c3e
[2024-04-03T21:22:48.749+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Chuks' AIRFLOW_CTX_DAG_ID='Chuksfx_ETL' AIRFLOW_CTX_TASK_ID='transform_response' AIRFLOW_CTX_EXECUTION_DATE='2024-04-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-02T00:00:00+00:00'
[2024-04-03T21:22:48.762+0000] {python.py:202} INFO - Done. Returned value was: None
[2024-04-03T21:22:48.769+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=Chuksfx_ETL, task_id=transform_response, execution_date=20240402T000000, start_date=20240403T212248, end_date=20240403T212248
[2024-04-03T21:22:48.818+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-03T21:22:48.829+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/models/baseoperator.py:1201: AirflowProviderDeprecationWarning: Call to deprecated class SnowflakeOperator. (This class is deprecated. Please use `***.providers.common.sql.operators.sql.SQLExecuteQueryOperator`. Also, you can provide `hook_params={'warehouse': <warehouse>, 'database': <database>, 'role': <role>, 'schema': <schema>, 'authenticator': <authenticator>,'session_parameters': <session_parameters>}`.)
  result = cls.__new__(cls)

[2024-04-03T21:22:48.842+0000] {taskinstance.py:3312} INFO - 1 downstream tasks scheduled from follow-on schedule check
